packages/web3/test/fixtures/system_test_utils.ts

```diff
@@ -0,0 +1,115 @@
+/*
+This file is part of web3.js.
```

Comment:
You do not need to copy this file. Just create a symlink (like packages/web3-eth/test/fixtures/system_test_utils.ts). the fastest way is just to copy symlink from another package and paste it to your package, but you should use Finder for this

Reply:
Just create a sym-link from root level `fixtures` directory or copy the link from some other package.